app/controllers/hq/orphans_controller.rb

```diff
@@ -3,4 +3,63 @@ class Hq::OrphansController < HqController
   def show
     @orphan = Orphan.find(params[:id])
```

Comment:
@razorcd Fair enough.
@gmunumel For example, `@orphan = Orphan.find(params[:id])` is used in the show, edit & update actions. To eliminate the duplication, we could do the following:
```ruby
class Hq::OrphansController < HqController
  before_action :set_orphan, only: [:show, :edit, :update]

  def show
  end

  def edit
    load_associations
  end
  .
  .

  private
    def set_orphan
      @orphan = Orphan.find(params[:id])
    end
end
```
A common criticism of such crosscutting concerns is that code can no longer be read in a linear fashion as execution jumps from the `before_action` at the top of the file to the method definition in the `private` section and finally to the controller action, where the `@orphan` instance variable magically appears.